files/build_templates/sonic_debian_extension.j2

```diff
@@ -114,6 +114,9 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python-dev     \
     python3-dev
 
+# Install j2cli for handling jinja template
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install j2cli
```

Comment:
it is better to specify a version == so that it won't break if j2cli upgrades.

Reply:
updated j2cli with a specific version